app/src/features/apiClient/screens/apiClient/components/views/components/response/TestsView/TestsView.tsx

```
@@ -82,18 +93,31 @@ export const TestsView: React.FC<TestsViewProps> = ({ testResults, handleTestRes
         </RQButton>
       </div>
 
-      <div className="test-results-list">
-        {filteredTestResults.map((testResult, index) => (
-          <TestResultItem key={index} testResult={testResult} />
-        ))}
-      </div>
+      { isResultRefreshing ? (
+        <div className="test-results-list">
+          {filteredTestResults.map((testResult, index) => (
+            <div className="test-result-item">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)